components/wrap-collection-layout.js

```diff
@@ -6,14 +6,16 @@ import { LoadMoreCollectionStories } from './load-more-stories-base';
 import { LazyLoadImages } from './responsive-image';
 import { ClientSideOnly } from './client-side-only';
 
-function addLoadMore(Component, data, slug, numStoriesToLoad) {
-  return React.createElement(LoadMoreCollectionStories, {
-    template: Component,
-    collectionSlug: slug,
-    params: { 'item-type': 'story' },
-    data: data,
-    numStoriesToLoad,       
-  });
+function maybeWrapLoadMore(Component, data, enable_load_more_button, slug, numStoriesToLoad) {
```

Comment:
Do an alias to camelcase the underscore names

`enable_load_more_button` to `enable_load_more_button: enableLoadMoreButton` and use it